runtime/doc/autocmd.txt

```diff
@@ -994,7 +994,7 @@ expect.
 Environment variables can be used in a pattern: >
 	:autocmd BufRead $VIMRUNTIME/doc/*.txt  set expandtab
 And ~ can be used for the home directory (if $HOME is defined): >
-	:autocmd BufWritePost ~/.vimrc   so ~/.vimrc
+	:autocmd BufWritePost ~/.config/nvim/init.vim   so <afile>
```

Comment:
I don't think it's necessary to be so specific. It is understood that `~/.config` means "what ~/.vim used to mean". And for new users, also, it is easier to understand what `~/.config` than an env var from the linux desktop standard.